ios/FluentUI.Demo/FluentUI.Demo/Demos/ButtonDemoController.swift

```diff
@@ -18,6 +18,10 @@ class ButtonDemoController: DemoController {
             let button = Button(style: style)
             button.setTitle("Button", for: .normal)
 
+            if style == .borderless {
+                button.contentEdgeInsets = .zero
```

Comment:
2 question: has this been already talked with design to make sure zero inset is intentional? also, shouldn't this be handle in the enum ButtonStyle's contentEdgeInsets variable return value?

Reply:
So, the idea is that if the calling code is setting the button's contentEdgeInsets, it should not be overriden by the style...
This is an example of how the "Account Details" button was before and after I tapped it (while setting the contentEdgeInsets to zero):

Before:
![before_tap](https://user-images.githubusercontent.com/68076145/87472279-8860b800-c5d4-11ea-89d5-73a0623fe240.png)

After:
![after_tap](https://user-images.githubusercontent.com/68076145/87472296-8d256c00-c5d4-11ea-8dbe-f617f2958246.png)